.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,68 @@
+name: Release notes
+
+on:
+  push:
+    branches:
+      - '!*'
+    tags:
+      - 'v*'
+
+jobs:
+  create_release_draft:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Drafting release
+          id: release_drafter
+          uses: release-drafter/release-drafter@v5
+          with:
+            config-name: release-drafter.yml
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Not sure if `secrets.RELEASE_NOTES_PAT` will be needed or not. We'll see...

Reply:
Ok